tests/phpunit/tests/functions/get_num_queries.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/**
+ * @group functions.php
+ * @covers ::get_num_queries
+ */
+class Tests_Functions_get_num_queries extends WP_UnitTestCase {
+
+	public function test_wp_get_num_queries() {
```

Comment:
```suggestion
	/**
	 * @ticket 54490
	 */
	public function test_wp_get_num_queries() {
```

Test methods should indicate the ticket that added them.